### PR TITLE
Pin conda build to deal with selectors failing

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: scripts/update_teams.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 #  - conda-smithy
 #  - pygithub 1.*
 #  - six
-#  - conda-build
+# This needs to be pinned because selectors are an issue in 1.20.2
+#  - conda-build =1.20.1
```
Comment (ericdill) [inline scripts/update_teams.py]: @jakirkham is this one (`=`) or two (`==`) to pin it?

Comment (jakirkham) [inline scripts/update_teams.py]: Might not need any `=` at all.

Comment (ericdill) [inline scripts/update_teams.py]: Also can I even leave comments in the conda execute header?

Comment (ericdill) [inline scripts/update_teams.py]: Local testing seems to indicate no

**Comment by jakirkham** [comment]
Cool, let's try this.

**Comment by ericdill** [comment]
This is a fun game

**Comment by msarahan** [comment]
I'm hoping to tag a new release tonight.  Builds might not be out until tomorrow, though.  If anyone is interested, I'll post a message in the gitter channel when the proposed version is ready to be tested.

**Comment by msarahan** [comment]
Beta tag at https://github.com/conda/conda-build/releases/tag/1.20.3beta - please test.  If all is well, I will release tomorrow.

**Pull request merged**